stl/src/xtime.cpp

```diff
@@ -86,12 +87,22 @@ _CRTIMP2_PURE long long __cdecl _Query_perf_counter() { // get current value of
     return li.QuadPart;
 }
 
-_CRTIMP2_PURE long long __cdecl _Query_perf_frequency() { // get frequency of performance counter
+static long long __cdecl _Query_perf_frequency_internal() { // get frequency of performance counter

```

Comment:
Why is this a separate function instead of being inlined into its single callsite?

Reply:
inlined